core/class/atlas.class.php

```diff
@@ -120,7 +120,7 @@ public static function recoveryUsbMount($devusb = '/dev/sda') {
     }
     shell_exec('sudo umount /mnt/usb');
     log::add(__CLASS__, 'debug', 'FSDISK -d');
-    config::save('migrationTextfine', __('Vérification de l\'espace de stockage.', __FILE__));
+    config::save('migrationTextfine', __('Vérification de l'espace de stockage.', __FILE__));
```

Comment:
Consider using a localization or internationalization library to handle all the text in your application. This will make it easier to manage and update the text, and will also allow you to support multiple languages in the future. [medium]